models/image_recognition/tensorflow/resnet50/fp32/eval_image_classifier_inference.py

```diff
@@ -72,12 +72,17 @@ def __init__(self):
                                  'the benchmark will use random/dummy data.',
                             dest="data_location", default=None)
 
+    arg_parser.add_argument('-i', "--iteration",
```

Comment:
Can you rename this arg to be `--steps` so that it matches the Int8 model (https://github.com/IntelAI/models/blob/master/models/image_recognition/tensorflow/resnet50/int8/benchmark.py#L91)

Reply:
@dmsuehir done.